docker/ci/docker-compose.yml

```diff
@@ -0,0 +1,36 @@
+version: "3.4"
+services:
+  armadillo:
+    hostname: armadillo
+    ports:
+      - 8080:8080
+    extra_hosts:
+      - "host.docker.internal:host-gateway"
+    image: molgenis/molgenis-armadillo:latest
+    environment:
+      LOGGING_CONFIG: 'classpath:logback-file.xml'
+      AUDIT_LOG_PATH: '/app/logs/audit.log'
+      SPRING_SECURITY_USER_PASSWORD: 'admin'
+    volumes:
+      - ${PWD}/fake-tree/armadillo/logs:/logs
+      - ${PWD}/fake-tree/armadillo/data:/data
+      - ${PWD}/fake-tree/armadillo/config:/config
```

Comment:
This works on MacOS and probably on Linux but not on CircleCI ... see `.circleci/config.yml` using `docker cp`